ballerina/vector_store.bal

```diff
@@ -122,10 +128,12 @@ public isolated class VectorStore {
             string gqlQuery = string `{
                 Get {
                     ${self.config.collectionName}(
-                        limit: ${self.topK}
+                        ${query.topK > -1 ? string `limit: ${query.topK}` : string ``}
```

Comment:
Do we have to handle the zero case separately? what happens if we send a zero for topK?

Reply:
Fixed in [3ece0b3](https://github.com/ballerina-platform/module-ballerinax-ai.weaviate/pull/8/commits/3ece0b35e6f6c6323ae78f74ac6d2b0a77462572)